src/MDFRobotoFontLoader.h

```diff
@@ -88,4 +88,17 @@
 
 /** Returns an italic version of the specified font. */
 + (nonnull UIFont *)italicFontFromFont:(nonnull UIFont *)font;
+
+/**
+ Whether a particular font would be considered "large" for the purposes of calculating
+ contrast ratios.
+
+ Large fonts are defined as greater than 18pt normal or 14pt bold. If the passed font is nil, then
+ this method returns NO.
+
+ @param font The font to examine, or nil.
+ @return YES if the font is non-nil and is considered "large".
+ */
```

Comment:
Please link out to something (the W3C spec?) that explains what this is all about.